.teamcity/components/generated/services.kt

```diff
@@ -38,6 +38,7 @@ var services = mapOf(
         "hdinsight" to "HDInsight",
         "hpccache" to "HPC Cache",
         "hsm" to "Hardware Security Module",
+        "healthbot" to "Health Bot",
```

Comment:
Should this be part of the healthcase or bot service?

Reply:
or will the be a bunch of new resources here?

Reply:
The swagger spec sits [here](https://github.com/Azure/azure-rest-api-specs/tree/master/specification/healthbot/resource-manager
). So I guess It's not part of healthcare or bot service.

Only one resource is defined in the spec.

Reply:
given its basically bot/chat ai like the bot service it would make sense to put it under bots?

Reply:
I've made changes according to your comments please help to review the the changes. thanks.